timemachine/fe/free_energy.py

```diff
@@ -392,12 +428,65 @@ def _combine(self, ligand_values, host_values=None):
         return np.concatenate([host_values, ligand_values])
 
 
-def get_context(initial_state: InitialState) -> Context:
-    bound_impls = [p.to_gpu(np.float32).bound_impl for p in initial_state.potentials]
+def get_water_sampler_params(initial_state: InitialState) -> NDArray:
```

Comment:
Is it possible to add a unit test for this? (Using initial states emitted from the SingleTopology/RBFE logic?) i.e. assert that the w-coords etc. for dummy atoms at the two end-states are being read-off correctly

Reply:
Added a test in https://github.com/proteneer/timemachine/pull/1286/commits/8ed067899ec3fcef0d619e486ad7e91a141032b2